arangodb-net-standard/AdminApi/AdminApiClient.cs

```diff
@@ -220,9 +220,9 @@ public virtual async Task<GetLicenseResponse> GetLicenseAsync(CancellationToken
                 if (response.IsSuccessStatusCode)
                 {
                     var stream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false);
-                    return DeserializeJsonFromStream<GetLicenseResponse>(stream);
+                    return await DeserializeJsonFromStreamAsync<GetLicenseResponse>(stream);
```

Comment:
We need to call `ConfigureAwait(false)` here as well, don't we? The underlying operation for the default serializer isn't really async, but that may not be the case for other serializer implementations.

I think we either need to add `ConfigureAwait(false)` 
```
return await DeserializeJsonFromStreamAsync<GetLicenseResponse>(stream).ConfigureAwait(false);;
```
or return the task directly without awaiting:
```
return DeserializeJsonFromStreamAsync<GetLicenseResponse>(stream);
```

I vote for returning the task directly if we don't need to `await`, which usually saves the need to create a state machine behind the scenes (but can be considered a micro optimization at this point).

Reply:
I've added `ConfigureAwait(false)` to the `DeserializeJsonFromStreamAsync()` calls.